controllers/apply/form-router-next/submission.js

```diff
@@ -88,7 +88,7 @@ module.exports = function(
             /**
              * Store submission in salesforce if enabled
              */
-            if (enableSalesforceConnector === true && !appData.isTestServer) {
```

Comment:
Removed this so we can have Cypress push to Salesforce locally, will reinstate after testing and before merging this PR.